### PR TITLE
Fix custom renderer

### PR DESCRIPTION
**Pull request by martinRenou**

### CODE REVIEW

**Comment by martinRenou** [comment]
@maartenbreddels would you mind checking this PR? :)

File: js/src/extension.ts [hunk 1]
```diff
@@ -2,19 +2,25 @@
 // It contains some requirejs configuration and the `load_ipython_extension`
 // which is required for any notebook extension.
 
+import * as Handsontable from 'handsontable/dist/handsontable.full.js';
+import 'handsontable/dist/handsontable.min.css';
+import '../../src/custom.css';
+
```
Comment (maartenbreddels) [inline js/src/extension.ts]: This will give an issue with the renderer extension, since you want only 1 handsontable (both extensions the same) we need to make it into a separate module.

**Comment by maartenbreddels** [comment]
I need to read better :)
Anyway, the idea of the separate module was to make sure only 1 handsontable gets loaded, how come there are two?

**Comment by martinRenou** [comment]
@maartenbreddels 
So basically, in [this PR](https://github.com/QuantStack/ipysheet/pull/25) I made some changes on the webpack config. After those changes `renderer` and `index` targets were both shipping their own `Handsontable`. And this broke the custom renderers.

So in this PR I fixed it, and `Handsontable` is once again an external dependency for the `index` target and the `renderer` target. So now it works fine.

I'm fine for putting back the `handsontable` target if it was important. I'm not sure why I removed it in the end. I think my issue was that TypeScript was changing the 
```javascript
import * as Handstontable from 'handsontable'
``` 
into 
```javascript
require('../lib/handsontable.js')'
``` 
due to the `handsontable.js` file. And webpack could not resolve the external dependency anymore.

File: js/src/extension.ts (hunk 1 above)
Comment (martinRenou) [reply js/src/extension.ts]: I have only one `handsontable`, this one :)

**Comment by maartenbreddels** [comment]
I'd rather have it as it was since with this PR Handsontable is always loaded (even if ipysheet is not used).

**Comment by martinRenou** [comment]
Ok, I'm not sure it worked due to TypeScript. But I can try

File: js/src/extension.ts [hunk 2]
```diff
@@ -8,8 +8,7 @@ if (window['require'] !== undefined) {
         map: {
             "*" : {
                 "ipysheet": "nbextensions/ipysheet/index",
-                "handsontable": "nbextensions/ipysheet/handsontable",
-                "handsontable/dist/handsontable.full.js":  "nbextensions/ipysheet/handsontable",
```
Comment (martinRenou) [inline js/src/extension.ts]: Why was this needed?

File: js/src/renderer.ts [hunk 3]
```diff
@@ -1,7 +1,8 @@
 import * as widgets  from '@jupyter-widgets/base';
 import {extend} from 'lodash';
 import * as pkg from '../package.json';
-import {Handsontable} from './handsontable';
+// @ts-ignore
```
Comment (martinRenou) [inline js/src/renderer.ts]: Ignoring with TypeScript (otherwise TypeScript complains that it does not exist)

**Comment by martinRenou** [comment]
@maartenbreddels Ok done :) Hope you are happy with the changes. Works like a charm now!

**Comment by maartenbreddels** [comment]
Why is the _prefix needed?

**Comment by martinRenou** [comment]
There was a conflict between `'./node_module/handsontable'` and `'handsontable'` for TypeScript. But now that I ignore the Handsontable variable in TypeScript it is fine anyway and it does not check the `require`. So I removed the _ prefix
